.github/ISSUE_TEMPLATE/package_issue.yml

```diff
@@ -0,0 +1,84 @@
+name: 'Package issue'
+description: Report Package issues or unexpected behavior
+title: "[Issue]: <PackageIdentifier>"
+labels: ["Issue-Bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please create issues related to the winget.exe client here [microsoft/winget-cli](https://github.com/microsoft/winget-cli/issues/new/choose)
+  - type: checkboxes
+    attributes:
+      label: Pre-checks
+      description: Please go through these Pre-checks before submitting a issue.
+      options:
+        - label: I have search for my issue and not found a duplicate/resolved issue.
+          required: true
+        - label: I have not been informed if the issue is resolved in a preview version of the winget client.
+          required: true
+  - type: dropdown
+    id: issue
+    attributes:
+      label: Issue Type
+      description: What kind of issue are you running into?
+      options:
+        - Other
+        - Installation issue
+        - Side-By-Side installation
+        - No applicable update found.
+        - Different install technology
+        - Multiple installed packages found
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Brief description of your issue
+      placeholder: Briefly describe your issue here.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      placeholder: How to reproduce the issue?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Actual behavior
+      placeholder: What is currently happening?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      placeholder: What did you expect to happen?
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Environment
+      description: Please copy and paste your winget --info output here.
+      placeholder: |
+        [winget --info]
+        Windows Package Manager version
+        Windows: Windows.Desktop version
+        Package: Microsoft.DesktopAppInstaller version
+
+        Any other software?
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
```

Comment:
Issue forms still do not support this:

<details>
<summary> Surprise! </summary>

```
# 37 days left until 1st April 
function Write-Console { 
   Write-Output "April Fool."
}
Write-Console
```
</details>

So, we can ask the user to attach log files along with screenshots.

Reply:
Yeah i was looking for that so we could toggle the log/screenshots :(
Maybe just change the Screenshot section to files/log dump? Because the log section can still be very much used depending on what output you post
For example this https://github.com/OfficialEsco/winget-pkgs/issues/6 contains the whole log for Honeygain.Honeygain which is not too messy, and if someone knows what they are looking for they can post the last 14+- lines instead of the full file

Reply:
Please take a look at: https://github.com/OfficialEsco/winget-pkgs/pull/7

Reply:
Thank you.